answer.json

```
@@ -209,24 +194,6 @@
         ],
         "answer": "No, Cofl is not ratted, you are welcome to decompile the mod from <#890682907889373257> and see for yourself."
```

```suggestion
        "answer": "No, Cofl is not ratted, you are welcome to decompile the mod from <#890682907889373257> and see for yourself. Or use https://isthisarat.com"
```
a lot of people don't know how to decompile